### PR TITLE
pre-merge review comments addressed: framework/DataObjects

### PR DESCRIPTION
**Pull request by PaulTalbot-INL**
Addresses comments on modules in `raven/framework/DataObjects` relating to the merge of the `dataobject-rework` branch into `devel`.

The only comment that should be resolved before merge that wasn't resolved is the question of merging the DataObject into the DataSet.  So far the prevailing opinion is to leave them separate, so as to leave room of other data entities in the future.

### CODE REVIEW

**Comment by PaulTalbot-INL** [comment]
It looks like all the comments weren't done yet, so this isn't ready for merge yet.

File: framework/DataObjects/DataSet.py [hunk 1]
```diff
@@ -776,12 +772,16 @@ def _checkRealizationFormat(self,rlz):
         self.raiseAWarning('Variable "{}" is not an acceptable type: "{}"'.format(key,type(value)))
         return False
       # check if index-dependent variables have matching shapes
-      # FIXME: this check will not work in case of variables depending on multiple indexes. When this need comes, we will change this check(alfoa)
+      # FIXME: this check will not work in case of variables depending on multiple indexes.
+      #    When this need comes, we will change this check(alfoa)
       if self.indexes:
         if key in self._fromVarToIndex and rlz[self._fromVarToIndex[key]].shape != rlz[key].shape:
-          self.raiseAWarning('Variable "{}" has not a consistent shape with respect its index "{}": shape({}) /= shape({})!'.format(key,self._fromVarToIndex[key],rlz[key].shape,rlz[self._fromVarToIndex[key]].shape))
+          self.raiseAWarning(('Variable "{}" has not a consistent shape with respect its index "{}": '+\
```
Comment (derekstucki) [inline framework/DataObjects/DataSet.py]: The backslash at the end is not needed since this is already in parens.

**Comment by alfoa** [comment]
news?

**Comment by PaulTalbot-INL** [comment]
The `PostProcessors\TemporalDataMiningPostProcessor\DimensionalityReduction.MultiDimensionalScaling` test is failing due to timeout.  Is this a new phenomenon?

**Comment by alfoa** [comment]
loooks like

**Comment by alfoa** [comment]
Considering Windwos machine is down.... we merge this

**Pull request merged**